### PR TITLE
Fixing small issues

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] Update

**What category or categories does it affect?**

_Multiple_

**What?**

_Adding new scopes to GHG protocol refrigerant gases, fixing the cradle_to_gate typo in BEIS data, and revising one waste category._

**Why?**

_To increase the consistency in the OEFDB_

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
-Waste,Glass Waste,waste_type_mixed_food_and_organic_garden-disposal_method_landfill,Mixed food and garden waste disposal (organic) - landfill,tonne,not-supplied,587.344,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,gate_to_grave,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for mixed food and garden waste disposal (organic) - landfill. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Waste,Refuse Waste,waste_type_household_residual_waste-disposal_method_anaerobic_digestion,Household residual waste disposal - anaerobic digestion,tonne,not-supplied,8.951,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for household residual waste disposal - anaerobic digestion. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5055]: Not sure we need a new category here - how does this differ from "General Waste"?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
+Waste,Refuse Waste,waste_type_household_residual_waste-disposal_method_landfill,Household residual waste disposal - landfill,tonne,not-supplied,446.242,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,gate_to_grave,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for household residual waste disposal - landfill. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Waste,Refuse Waste,waste_type_household_residual_waste-disposal_method_open_loop,Household residual waste disposal - open-loop,tonne,not-supplied,21.294,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for household residual waste disposal - open-loop. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5060]: This definitely needs to change (from glass) - but again think this is just "General Waste"?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
+Waste,Refuse Waste,waste_type_mixed_food_and_organic_garden-disposal_method_combustion,Mixed food and garden waste (organic) - combustion,tonne,not-supplied,21.294,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for mixed food and garden waste (organic) - combustion. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5061]: I think this one should be category "Food and Organic Waste"?

File: OpenEmissionFactorsDB.csv [hunk 4]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
+Waste,Refuse Waste,waste_type_mixed_food_and_organic_garden-disposal_method_anaerobic_digestion,Mixed food and garden waste disposal (organic) - anaerobic digestion,tonne,not-supplied,8.951,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for mixed food and garden waste disposal (organic) - anaerobic digestion. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5062]: I think this one should be category "Food and Organic Waste"?

File: OpenEmissionFactorsDB.csv [hunk 5]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
+Waste,Refuse Waste,waste_type_mixed_food_and_organic_garden-disposal_method_composting,Mixed food and garden waste disposal (organic) - composting,tonne,not-supplied,8.951,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,end_of_life,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for mixed food and garden waste disposal (organic) - composting. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5063]: I think this one should be category "Food and Organic Waste"?

File: OpenEmissionFactorsDB.csv [hunk 6]
```diff
@@ -5035,33 +5035,33 @@ Waste,General Waste,waste_type_carpet-disposal_method_landfilled,Carpet - Landfi
...
+Waste,Refuse Waste,waste_type_mixed_food_and_organic_garden-disposal_method_landfill,Mixed food and garden waste disposal (organic) - landfill,tonne,not-supplied,587.344,not-supplied,not-supplied,not-supplied,not-supplied,not-supplied,,3,gate_to_grave,BEIS,2021,2021,,GB,,Climatiq,2022/03/20,Emission intensity for mixed food and garden waste disposal (organic) - landfill. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:5064]: I think this one should be category "Food and Organic Waste"?

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 1 above)
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:5055]: It is the category defined by the source (BEIS). Let's discuss if we want to stick with the source category or not? @patstirling

**Review by patstirling — APPROVED** [review]

**Pull request merged**